atlite/datasets/tz_era5.py

```diff
@@ -256,6 +268,21 @@ def retrieve_raw_data(
     x = retrieval_params["x"]
     y = retrieval_params["y"]
 
+    # Check if year is in the list of complete weather years
```

Comment:
This is the only real change, the rest is just linting/deleting stuff to comply with pre-commit